src/seedu/addressbook/AddressBook.java

```diff
@@ -200,11 +200,29 @@
      */
     public static void main(String[] args) {
```

Comment:
Hope you ran tests after inlining functions.
Can you inline only methods to make code harder to read and navigate?